ui/src/mockState.tsx

```diff
@@ -25,7 +26,8 @@ const localState = {
       duration: '15m',
     },
   ],
-  variables: initialState(),
+  variables: initialVariablesState(),
```

Comment:
would we need to worry about sensitive data in variables?

Reply:
the initial state of variables doesnt have much in it. plus this is just mocks